### ISSUE TITLE
Create simple interface to search for keywords and display corresponding images

### ISSUE THREAD

**Issue by stumpdk** [stumpdk/CultureEvacuator2#15]

**Comment by jacobriddersholm** [comment]
Say what?! Hvad laver du? ;)

**Comment by stumpdk** [comment]
Ja, stenede lidt på Gamle København i går, og ville lige se om vores harvester virker. Den har masser af potentiale!


Sendt fra Outlook Mobile






On Thu, Jan 21, 2016 at 3:13 PM -0800, "jacobriddersholm" <notifications@github.com> wrote:





Say what?! Hvad laver du? ;)

---

**Comment by jacobriddersholm** [comment]
Cool. Jeg er stadig deadset på at implementerer den her på Natmus.

Kanske vi skulle prøve at skaffe dollars til et joint venture... ?
